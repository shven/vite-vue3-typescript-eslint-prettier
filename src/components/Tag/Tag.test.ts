import { mount } from '@vue/test-utils';
import Component from './VueTag.vue';

describe('VueTag.vue', () => {
    const wrapper = mount(Component, {
        props: {
            text: 'tag text'
        }
    });

    it('Match snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
