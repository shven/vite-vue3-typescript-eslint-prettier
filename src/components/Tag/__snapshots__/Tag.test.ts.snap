// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VueTag.vue > Match snapshot 1`] = `
<div
  class="tag"
  data-v-d15b973d=""
>
  tag text
</div>
`;
